src/ima.js

```diff
@@ -399,7 +399,6 @@ export default class Ima extends BasePlugin {
     this._sdk.settings.setPlayerVersion(VERSION);
     this._sdk.settings.setVpaidAllowed(true);
     this._sdk.settings.setVpaidMode(this._sdk.ImaSdkSettings.VpaidMode.ENABLED);
-    this._sdk.settings.setDisableCustomPlaybackForIOS10Plus(true);
```

Comment:
@dan-ziv if we have playsinline config then this is required, so let's make this somewhere in the form of:
`this._sdk.settings.setDisableCustomPlaybackForIOS10Plus(config.playsinline);`

need to make sure playback preformance with regards to the user gesture restriction in this case as well

Reply:
@OrenMe 
From my checks this flag is not mandatory and IOS10Plus playback works just fine

Reply:
this is just if we want to disable AV player full screen on iOS10Plus - do we want?

Reply:
this is what I meant in my comment - if player is set to work inline iOS10 then we need to set this as well, so need to align to player config, and also enable passing other options if we are on the subject, like the vpaidAllowed and vpaidMode.

Reply:
no need to set this as well, playsinline on ios10plus works without this flag